### ISSUE TITLE
Error: Invalid TF_Status: 2 Message: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize

### ISSUE THREAD

**Issue by chanduthedev** [Infinitay/face-rec-photo-gallery#10]
Hi @Infinitay,

I am getting `Error: Invalid TF_Status: 2 ` error when I run face-api.js in my GPU machine. I see your comments in https://github.com/tensorflow/tfjs/issues/2760 issue saying you resolved this error. Can you please help me on this?

I couldn't find any links or docs to enable GPU options in NodeJS. I already raised issue https://github.com/tensorflow/tfjs/issues/3533 in tensor flow.

My apologies for raising unrelated issue in this repo.

Thanks in advance.

**Comment by chanduthedev** [comment]
@Infinitay I managed solve this issue.  Need to export the value as below like below.
`export TF_FORCE_GPU_ALLOW_GROWTH=true`

**Comment by Infinitay** [comment]
My mistake, I had the following in another private repository that uses the same code here as it's backbone with the solution you stated.

For what it's worth in my experience, I sometimes also ended up getting `Error: Invalid TF_Status: 2` even with explicitly stating `TF_FORCE_GPU_ALLOW_GROWTH=true`. I believe it was due to the source input being too large/too many tensors (?).

So just a heads up. However, for the most part that should do the trick and solve your issue. 

EDIT: On the off chance you too start getting the same issue once more despite setting growth to true, feel free to do what I did and start clearing the tensors from memory to help out.

Mind you I am using a GTX 970 with 3.5GB of VRAM
